src/api/traits.h

```diff
@@ -0,0 +1,50 @@
+#ifndef HERMES_TRAITS_H
+#define HERMES_TRAITS_H
+
+#include <unordered_map>
+
+#include "hermes_types.h"
+
+namespace hermes {
+namespace api {
+struct BlobInfo {
+  std::string bucket_name;
+  std::string blob_name;
+};
+typedef BlobInfo TraitInput;
+// typedef void* TraitCallback;
+
+struct Trait;
+typedef std::function<void(TraitInput &, Trait *)> TraitCallback;
+// typedef void(*TraitCallback)(TraitInput &, Trait *);
```

Comment:
Please remove commented out code, or add a comment explaining why it's left in.

Reply:
Done